examples/makefile

```diff
@@ -3,8 +3,8 @@ all: setup network test
 # --- Configurable parameters
 
 APPROACHES			= anisotropic
-MODEL				= steady_turbine
-NUM_TRAINING_CASES	= 100
+MODEL				= burgers
+NUM_TRAINING_CASES	= 1
```

Comment:
This is okay to start off with. However, further down the line I think it would be a good idea to use a larger number of cases.